ecommerce_worker/fulfillment/v1/tasks.py

```
@@ -21,14 +21,15 @@ def fulfill_order(self, order_number):
         None
     """
     ecommerce_api_root = get_configuration('ECOMMERCE_API_ROOT')
-    worker_access_token = get_configuration('WORKER_ACCESS_TOKEN')
     max_fulfillment_retries = get_configuration('MAX_FULFILLMENT_RETRIES')
+    signing_key = get_configuration('JWT_SECRET_KEY')
+    issuer = get_configuration('JWT_ISSUER')
+    service_username = get_configuration('ECOMMERCE_SERVICE_USERNAME')
 
-    if not (ecommerce_api_root and worker_access_token and max_fulfillment_retries):
+    if not all((ecommerce_api_root, max_fulfillment_retries, signing_key, issuer, service_username)):
```

Can you make `get_configuration` raise an error if the values are missing?